archean/egg-archean.json

```diff
@@ -0,0 +1,174 @@
+        "update_url": null
+    },
+    "exported_at": "2024-10-21T00:55:52+00:00",
+    "name": "Archean",
+    "author": "tueem@tomatentum.net",
+    "uuid": "01555bd3-155f-4ab6-b63d-c8bed0a3585f",
+    "description": "Archean invites you to unleash your creativity alone or with friends in a creative sandbox mode.Design and build vehicles, bases, rockets, space stations and much more using customizable modular blocks.Create unique configurations using components, program their behavior and explore an environment free of constraints.\\nTest your builds, refine your designs and defy the laws of physics.Archean is a game in development focused on engineering and build.",
+    "features": [],
+    "docker_images": {
+        "Ubuntu": "ghcr.io\/pelican-eggs\/yolks:ubuntu"
+    },
+    "file_denylist": [],
+    "startup": ".\/archean_start.sh",
+    "config": {
+        "files": "{}",
+        "startup": "{\n\"done\": \"Server started\"\n}",
+        "logs": "{}",
+        "stop": "^C"
+    },
+    "scripts": {
+        "installation": {
+            "script": "#!\/bin\/bash\n\nmv \/Archean-server-alpha \/mnt\/server\/Archean-server-alpha\n\napt update -y\napt install -y curl\ncurl -o \/mnt\/server\/archean_start.sh https:\/\/git.tomatentum.net\/tueem\/archean-server-pelican\/raw\/branch\/master\/archean_start.sh\nchmod -R 755 \/mnt\/server\/\n\necho ******************************************\necho Installation Completed\necho ******************************************",
+            "container": "xenon3d\/archean-server-alpha",
+            "entrypoint": "bash"
+        }
+    },
+    "variables": [
+        {
+            "sort": 1,
+            "name": "Server Name",
+            "description": "The servers name that is, among others, displayed on the public server page.",
+            "env_variable": "SERVER_NAME",
+            "default_value": "",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": [
+                "required",
+                "string",
+                "not_regex:\/\\\/\/"
+            ],
+            "field_type": "text"
+        },
+        {
+            "sort": 2,
+            "name": "Game Mode",
+            "description": "Changes the servers game mode between adventure and creative.",
+            "env_variable": "GAME_MODE",
+            "default_value": "adventure",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": [
+                "required",
+                "in:creative,adventure"
+            ],
+            "field_type": "text"
+        },
+        {
+            "sort": 3,
+            "name": "Online Mode",
+            "description": "Whether the server is publicly discoverable or not.",
+            "env_variable": "ONLINE_MODE",
+            "default_value": "0",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": [
+                "required",
+                "in:0,1"
```

Comment:
Unless something changed in Pelican that I'm unaware of, this should simply be changed to `boolean` (`required` can also be omitted if this is changed).